pages/_data/questionData.js

```diff
@@ -55,65 +55,72 @@ const getFlashCardQuestions = (questions) =>
   questions.filter((question) => !question['Multiple Choice Only']);
 
 const getUniqueCategories = (questions) => {
-  return [
-    ...new Set(
-      questions
-        .map((question) => question.Tags)
-        .flat()
-        .sort()
-    ),
-  ];
+	return [
+		'All Questions',
+		...new Set(
+			questions
+				.map((question) => question.Tags)
+				.flat()
+				.sort()
+		),
+	];
```

Comment:
There are a few complicated array actions here that I think are worth calling out, so I'll try to break it down. The end result that we want is an array with each unique category we find in alphabetical order (except for making sure "All Questions" is the first item).

- `...new Set(arrayWithDuplicateItems)` lets us [remove duplicate items](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#remove_duplicate_elements_from_an_array)
- You know about `.map`, but since `question.Tags` is an array, we end up with an array of arrays, like `[["ARIA"], ["Color Contrast", "Design"]]`. By using `.flat`, we take those inner arrays and [flatten them out](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) so we end up with a one-dimensional array, like `["ARIA", "Color Contrast", "Design"]`
- Combining all of these things with `.sort` gives us the list of categories that we need, All Questions, ARIA, Assistive Technologies, etc. as shown on the home page